marketdata/market/dex/dex.go

```diff
@@ -59,9 +59,9 @@ func normalizeTicker(price *CoinPrice, provider string) (*blockatlas.Ticker, err
 		return nil, errors.E(err, "normalizeTicker parse value24h error",
 			errors.Params{"PriceChange": price.PriceChangePercent, "Symbol": price.BaseAssetName})
 	}
-	tokenId := price.QuoteAssetName
+	tokenId := price.BaseAssetName
 	if tokenId == BNBAsset {
-		tokenId = price.BaseAssetName
+		tokenId = price.QuoteAssetName
```

Comment:
is there a way to add unit test for this?

Reply:
we already have, I need to fix that